qemu/tests/cfg/block_check_serial.cfg

```diff
@@ -0,0 +1,30 @@
+- block_check_serial:
+    type = block_check_serial
+    virt_test_type = qemu
+    kill_vm = yes
+    start_vm = yes
+    data_images="stg1 stg2 stg3"
+    images += ' ${data_images}'
+    force_create_image_image1 = no
+    force_create_image = yes
+    drive_format_stg1 = virtio
+    drive_format_stg2 = scsi-hd
+    drive_format_stg3 = scsi-hd
+    image_size_stg1 = 1G
+    image_size_stg2 = 2G
+    image_size_stg3 = 3G
+    image_name_stg1 = images/stg1
+    image_name_stg2 = images/stg2
+    image_name_stg3 = images/stg3
+    image_serial_stg1="stg1_serial_123456789a_123456789b_123456789b_123456789d_123456789e_end"
+    image_serial_stg2="stg2_serial_123456789a_123456789b_123456789b_123456789d_123456789e_end"
+    image_serial_stg3="stg3_device_123456789a_123456789b_123456789b_123456789d_123456789e_end"
+    blk_extra_params_stg1 = "serial=${image_serial_stg1}"
+    blk_extra_params_stg2 = "serial=${image_serial_stg2}"
+    blk_extra_params_stg3 = "device_id=${image_serial_stg3}"
+    Windows:
+        cmd = "powershell -command "get-disk|? {{($_.UniqueId -eq '{0}') -or "
+        cmd += "($_.SerialNumber -eq '{0}')}}|FT -HideTableHeaders"
+    Linux:
+        cmd = "sg_vpd {0} -p 0x80 |tail -n 1 > {1};sg_vpd {0} -p 0x83 |tail -n 1 >> {1};"
```

Comment:
Could we use a more accurate cmd name?
like: `cmd_get_disk_id`

Reply:
updated